src/VaultV2.sol

```diff
@@ -34,6 +34,9 @@ import {ISharesGate, IReceiveAssetsGate, ISendAssetsGate} from "./interfaces/IGa
 /// share price but not faster than the maxRate, and the interest are accrued only once per transaction. In order to
 /// protect against inflation attacks, the vault might need to be seeded with an initial deposit. See
 /// https://docs.openzeppelin.com/contracts/5.x/erc4626#inflation-attack
+/// @dev If they make the rate increase by a large factor, donations and forceDeallocate penalties can be in part stolen
```

Comment:
I don't like the word "stolen" here, because nothing says that those funds should go to the current depositors (and not future depositors, even if we could argue that they are "opportunistic")